commands/acl-genpass.md

```diff
@@ -7,11 +7,11 @@ uses a weaker system that is likely still better than picking a weak password
 by hand.
 
 By default (if /dev/urandom is available) the password is strong and
-can be used for other uses in the context of a Redis application, for
+can be used for other uses in the context of a Valkey application, for
 instance in order to create unique session identifiers or other kind of
 unguessable and not colliding IDs. The password generation is also very cheap
 because we don't really ask /dev/urandom for bits at every execution. At
-startup Redis creates a seed using /dev/urandom, then it will use SHA256
+startup Valkey creates a seed using /dev/urandom, then it will use SHA256
```

Comment:
Would "server" be clear enough here? if uncertain, we can keep valkey

```suggestion
startup, the server creates a seed using /dev/urandom. Then it will use SHA256
```

Reply:
I think the higher level goal here is to remove "redis".